common/rabbitmq/values.yaml

```diff
@@ -82,12 +86,28 @@ alerts:
   # Configurable service label of the alerts. Defaults to `.Release.Name`.
   # service:
 
-kind: deployment  # needs to be statefulset with persistence.enabled and ...
-replicas: 1       # replicas more than one. Clustering is currently not implemented
+# replicas. Uses statefulset for multiple replicas.
+# Cluster formation is enabled with cluster.enabled = true.
+replicas: 1
+cluster:
+  enabled: false
+  ha: true # Enables HA
+  erlangCookie: "DefaultErlangSecretCookie"
```

Comment:
I would prefer an empty default, and failing installation than quietly using a insecure default.